Gulpfile.js

```diff
@@ -34,6 +34,7 @@ var childProcess         = require('child_process');
 var listBrowsers         = require('testcafe-browser-tools').getInstallations;
 var npmAuditor           = require('npm-auditor');
 var checkLicenses        = require('./test/dependency-licenses-checker');
+var sourcemaps           = require('gulp-sourcemaps');
```

Comment:
Use ES6+ syntax everywhere:
* `const` and `let` instead of `var`
* arrow functions
* etc